schema.sql

```diff
@@ -19,3 +19,20 @@ CREATE TABLE authors(
     last_name VARCHAR(50),
     PRIMARY KEY(id)
 );
+
+CREATE TABLE music_album (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    genre VARCHAR(25),
+    author VARCHAR(50),
+    label VARCHAR (50),
+    publish_date  DATE,
+    on_spotify BOOLEAN, 
+    archived BOOLEAN,
+    PRIMARY KEY(id)
+);
```

Comment:
Great job including the properties from the item class in the music_album table.
Please help me by adding some more properties to the table. We need to add the genre_id, author_id, and label_id.
Also please add the foreign keys with the references to the genres(id), authors(id), and labels(id)

Guide yourself with the `games` table implemented by Jose